src/Sylius/Behat/Page/User/RegisterPage.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\User;
+
+use Sylius\Behat\Page\SymfonyPage;
+
+/**
+ * @author Magdalena Banasiak <magdalena.banasiak@lakion.com>
+ */
+class RegisterPage extends SymfonyPage
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getRouteName()
+    {
+        return 'sylius_user_registration';
+    }
+
+    /**
+     * @param string $email
+     */
+    public function register($email)
+    {
+        $document = $this->getDocument();
+
+        $document->fillField('First name', 'Ted');
+        $document->fillField('Last name', 'Smith');
+        $document->fillField('Email', $email);
+        $document->fillField('Password', 'pswd');
+        $document->fillField('Verification', 'pswd');
+
+        $document->pressButton('Register');
+    }
+
+    /**
+     * @throws \Exception
+     */
+    public function wasRegistrationSuccessful()
+    {
+        $flashMessage = $this->getDocument()->find('css', '.alert-success');
+
+        if (null === $flashMessage) {
+            throw new \Exception('Account is not registered.');
+        }
```

Comment:
IMO this method should return `boolean`. `return null !== $flashMessage` and then in the context `expect($this->registerPage->wasRegistrationSuccessful)->toBe(true)` like [here](https://github.com/michalmarcinkowski/Sylius/blob/546beba34235673327463e43e9951a1595cadead/src/Sylius/Behat/Context/Ui/CustomerContext.php#L67-L67)